tsl/src/data_node.c

```diff
@@ -1185,16 +1184,31 @@ get_hypertable_data_node(Oid table_id, const char *node_name, bool owner_check,
 					(errcode(ERRCODE_TS_DATA_NODE_NOT_ATTACHED),
 					 errmsg("data node \"%s\" is not attached to hypertable \"%s\"",
 							node_name,
-							get_rel_name(table_id))));
+							get_rel_name(ht->main_table_relid))));
 		else
 			ereport(NOTICE,
 					(errcode(ERRCODE_TS_DATA_NODE_NOT_ATTACHED),
 					 errmsg("data node \"%s\" is not attached to hypertable \"%s\", "
 							"skipping",
 							node_name,
-							get_rel_name(table_id))));
+							get_rel_name(ht->main_table_relid))));
 	}
 
+	return hdn;
+}
+
+static HypertableDataNode *
+get_hypertable_data_node(Oid table_id, const char *node_name, bool owner_check, bool attach_check)
```

Comment:
I would prefer that this function share name with the other function but with, e.g., a suffix `_from_relid`. It is a bit confusing to just change the order of the words. So, the suggested names would be something like:

```
hypertable_data_node_get
hypertable_data_node_get_from_relid
```

Reply:
This is the original name before this PR. It is an issue with GH that it shows it as a new name.

Reply:
I updated names of newly added functions in this PR to specify by what they retrieve or match. I didn't update this static function as it was named such originally.

Reply:
Yes, the there was a function there previously that had this name, but you also introduced a _new_ public function with a different name but sharing the same basic functionality, So, _previously_ there was no inconsistency (since there was only on internal function) but _after_ this change there is an inconsistency. 

Just noting, you can keep as is.